static/js/dashboard.js

```diff
@@ -85,1162 +93,1411 @@ $(document).mouseup(function (e) {
+          timeRange: timeRange,
+          panels: localPanels,
+          refresh: dbRefresh,
+        }),
+        null,
+        2
+      )
+    );
+  });
+
+  //get dashboard data from database
+  $.ajax({
+    method: "get",
+    url: "api/dashboards/" + dbId,
+    headers: {
+      "Content-Type": "application/json; charset=utf-8",
+      Accept: "*/*",
+    },
+    dataType: "json",
+    crossDomain: true,
+  }).then(function (res) {
+  
+    console.log(JSON.stringify(res));
+    $(".dbSet-dbName").val(res.name);
+    $(".dbSet-dbDescr").val(res.description);
+    $(".dbSet-jsonModelData").val(JSON.stringify(JSON.unflatten(res), null, 2));
+  });
+
+  showGeneralDbSettings();
+  addDbSettingsEventListeners();
 }
 
 function showGeneralDbSettings() {
-    $('.dbSet-general').addClass('selected')
-    $('.dbSet-generalHTML').removeClass('hide');
+  $(".dbSet-general").addClass("selected");
+  $(".dbSet-generalHTML").removeClass("hide");
 
-    $('.dbSet-jsonModel').removeClass('selected');
-    $('.dbSet-jsonModelHTML').addClass('hide')
+  $(".dbSet-jsonModel").removeClass("selected");
+  $(".dbSet-jsonModelHTML").addClass("hide");
 }
 
 function showJsonModelDbSettings() {
-    $('.dbSet-general').removeClass('selected')
-    $('.dbSet-generalHTML').addClass('hide');
+  $(".dbSet-general").removeClass("selected");
+  $(".dbSet-generalHTML").addClass("hide");
 
-    $('.dbSet-jsonModel').addClass('selected');
-    $('.dbSet-jsonModelHTML').removeClass('hide')
+  $(".dbSet-jsonModel").addClass("selected");
+  $(".dbSet-jsonModelHTML").removeClass("hide");
 }
 
 function addDbSettingsEventListeners() {
-    $('.dbSet-general').on('click', showGeneralDbSettings);
-    $('.dbSet-jsonModel').on('click', showJsonModelDbSettings)
+  $(".dbSet-general").on("click", showGeneralDbSettings);
+  $(".dbSet-jsonModel").on("click", showJsonModelDbSettings);
 }
 
 function saveDbSetting() {
-    let trimmedDbName = $('.dbSet-dbName').val().trim();
-
-    if (!trimmedDbName) {
-        // Show error message using error-tip and popupOverlay
-        $('.error-tip').addClass('active');
-        $('.popupOverlay, .popupContent').addClass('active');
-        $('#error-message').text('Dashboard name cannot be empty.');
-        return;
-    }
+  let trimmedDbName = $(".dbSet-dbName").val().trim();
+  let trimmedDbDescription = $(".dbSet-dbDescr").val().trim();
+  if (!trimmedDbName) {
+    // Show error message using error-tip and popupOverlay
+    $(".error-tip").addClass("active");
+    $(".popupOverlay, .popupContent").addClass("active");
+    $("#error-message").text("Dashboard name cannot be empty.");
+    return;
+  }
 
-    $('.dbSet-dbName').val("");
-    $('.dbSet-dbDescr').val("");
-    $('.dbSet-jsonModelData').val("");
+  $(".dbSet-dbName").val("");
+  $(".dbSet-dbDescr").val("");
+  $(".dbSet-jsonModelData").val("");
 
-    dbName = trimmedDbName;
-    dbDescr = dbData.description;
 
+  dbName = trimmedDbName;
+  dbDescr = trimmedDbDescription;
```

Comment:
Assigning the updated description value from the input field to a variable 
and pushing it into database

`dbDescr = trimmedDbDescription;`